plugin.rb

```diff
@@ -151,4 +151,17 @@ def create_json(obj)
     mount ::DiscourseStaffNotes::Engine, at: "/staff_notes"
   end
 
+  Warning.class_eval do
```

Comment:
This is quite dangerous. If I accepted this, even if Staff Notes were disabled in multisite it would still be creating staff notes.

You should use `add_model_callback` for this. It will automatically wire it up safely and ensure that the plugin is enabled before it is used.